CHANGELOG.md

```diff
@@ -14,11 +14,16 @@ Please add to the relevant subsections under Unreleased below on every PR where
 ### Added
 
 - Display warning when Kubernetes pod fails to schedule by @TimH98 in https://github.com/wandb/wandb/pull/7576
+- Added `ArtifactCollection.save()` by @amusipatla-wandb in https://github.com/wandb/wandb/pull/7555
```

Comment:
This is user-facing, could you please elaborate a bit more on what it means?

Reply:
yeah definitely, I changed it based off of [this comment](https://github.com/wandb/wandb/pull/7555#discussion_r1589812151), does the original work?